services/skipchain/skipchain.go

```diff
@@ -456,6 +461,20 @@ func (s *Service) lenSkipBlocks() int {
 	return len(s.SkipBlocks)
 }
 
+// saves the actual identity
+func (s *Service) save() {
```

Comment:
Yes, and while we're at it, we could also 
- add a `Load`-method
- add a `Path`-method
- remove the path from the `sda.NewServiceFunc`

should I open an issue or include that in here?